skore/src/skore/sklearn/_comparison/report.py

```diff
@@ -0,0 +1,169 @@
+from __future__ import annotations
+
+import time
+from collections.abc import Iterable
+from copy import deepcopy
+from typing import Optional
+
+import joblib
+import numpy as np
+
+from skore.externals._pandas_accessors import DirNamesMixin
+from skore.sklearn._base import _BaseReport
+from skore.sklearn._estimator.report import EstimatorReport
+
+
+class ComparisonReport(_BaseReport, DirNamesMixin):
```

Comment:
From the demo today, I think the feedback was to allow the following.

```python
report = ComparisonReport.from_estimators(
    estimators=[est1, est2], X_train, y_train, X_test, y_test
)
report = ComparisonReport.from_reports(
    reports=[report1, report2], ...
)
```

`from_reports` is actually the current `__init__` while the `from_estimators` is something that should create the `EstimatorReport` and pass it to the `from_reports` afterwards.

Reply:
We can potentially differ at first.

Reply:
So let's differ